src/james.crypto.md5.clss.pas

```diff
@@ -52,13 +57,37 @@ implementation
 { TMD5Stream }
 
 function TMD5Stream.GetStream: IDataStream;
+{$IFNDEF FPC}
+  function getMd5HashString(Value: string): string;
```

Comment:
@nunopicado according to this [link](http://delphi.wikia.com/wiki/Delphi_Release_Dates), Delphi XE8 was released a 2 years ago so, we can move on with your idea.